gui/WorldView.cxx

```diff
@@ -621,6 +699,56 @@ void WorldView::updateCameras()
 }
 
 //-----------------------------------------------------------------------------
+void WorldView::updateAxes()
+{
+  QTE_D();
+
+  if (d->axesDirty)
+  {
+    // Compute bounds of visible actors
+    auto const props = d->renderer->GetViewProps();
+    vtkBoundingBox bbox;
+
+    double tb[6];
+    d->imageActor->GetBounds(tb);
+
+    props->InitTraversal();
+    while (auto const prop = props->GetNextProp())
+    {
+      // Skip the axes, ground and frustums representations, and any hidden actors
+      if (prop == d->cubeAxesActor.GetPointer()
+          || prop == d->groundActor.GetPointer()
+          || prop == d->cameraRep->GetActiveActor()
```

Comment:
This removes the cameras entirely, not just the frustums.  There is some value in being able to measure the distance that the cameras covered in their motion.  However, the size of the frustum is arbitrary and should not be included.  Basically, for each camera that is visible we should add the camera center (a single point) to the bbox bounds.